models/experimental/functional_unet/tt/model_preprocessing.py

```diff
@@ -38,104 +38,108 @@ def create_unet_input_tensors(
     return torch_input_tensor, ttnn_input_tensor
 
 
-def create_unet_model_parameters(model: unet_shallow_torch.UNet, input_tensor: torch.Tensor, groups: int, device):
-    assert groups == 1, "Only groups=1 is supported for now"
-
+def create_unet_model_parameters(
+    model: unet_shallow_torch.UNet, input_tensor: torch.Tensor, groups: int, device: ttnn.Device
+):
     parameters = infer_ttnn_module_args(model=model, run_model=lambda model: model(input_tensor), device=None)
     assert parameters is not None
     for key in parameters.keys():
         parameters[key].module = getattr(model, key)
 
-    parameters.p1["parallel_config_override"] = {
-        "grid_size": (8, 8),
-        "num_cores_nhw": 64,
-    }
-    parameters.p2["parallel_config_override"] = {
-        "grid_size": (8, 8),
-        "num_cores_nhw": 60,
-    }
-    parameters.p3["parallel_config_override"] = {
-        "grid_size": (8, 8),
-        "num_cores_nhw": 60,
-    }
-    parameters.p4["parallel_config_override"] = {
-        "grid_size": (8, 7),
-        "num_cores_nhw": 55,
-    }
```

Comment:
Don't need these anymore?

Reply:
They were required for the old max pool, I didn't think there was a way to pass them to the new one 🤔